<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>789</width>
    <height>660</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dact</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Entries</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Filter:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="filterLineEdit"/>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QListView" name="fileListWidget">
             <property name="selectionMode">
              <enum>QAbstractItemView::ExtendedSelection</enum>
             </property>
             <property name="uniformItemSizes">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QProgressBar" name="filterProgressBar">
             <property name="visible">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="statisticsGroupBox">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="title">
           <string>Statistics</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QFormLayout" name="statisticsLayout">
             <property name="fieldGrowthPolicy">
              <enum>QFormLayout::FieldsStayAtSizeHint</enum>
             </property>
             <property name="formAlignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
             </property>
             <item row="1" column="0">
              <widget class="QLabel" name="hitsDescLabel">
               <property name="text">
                <string>Hits:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLabel" name="hitsLabel">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="entriesLabel">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Entries:</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Tree</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="2">
           <widget class="QLineEdit" name="highlightLineEdit"/>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Highlight:</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="DactTreeView" name="treeGraphicsView" native="true"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_1">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Sentence:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="sentenceLineEdit"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>789</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="openAction"/>
    <addaction name="openDirectoryAction"/>
    <addaction name="separator"/>
    <addaction name="saveCorpus"/>
    <addaction name="pdfExportAction"/>
    <addaction name="printAction"/>
    <addaction name="separator"/>
    <addaction name="preferencesAction"/>
    <addaction name="separator"/>
    <addaction name="quitAction"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="focusFilterAction"/>
    <addaction name="focusHighlightAction"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="previousAction"/>
    <addaction name="nextAction"/>
    <addaction name="separator"/>
    <addaction name="fitAction"/>
    <addaction name="zoomInAction"/>
    <addaction name="zoomOutAction"/>
    <addaction name="separator"/>
    <addaction name="previousTreeNodeAction"/>
    <addaction name="nextTreeNodeAction"/>
    <addaction name="separator"/>
    <addaction name="showFilterWindow"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="showMacrosWindow"/>
    <addaction name="showStatisticsWindow"/>
    <addaction name="downloadAction"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="helpAction"/>
    <addaction name="aboutAction"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="previousAction"/>
   <addaction name="nextAction"/>
   <addaction name="separator"/>
   <addaction name="zoomInAction"/>
   <addaction name="zoomOutAction"/>
   <addaction name="fitAction"/>
   <addaction name="previousTreeNodeAction"/>
   <addaction name="nextTreeNodeAction"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="quitAction">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="fitAction">
   <property name="icon">
    <iconset resource="../resources/dact.qrc">
     <normaloff>:/icons/viewmagfit.png</normaloff>:/icons/viewmagfit.png</iconset>
   </property>
   <property name="text">
    <string>Fit</string>
   </property>
   <property name="toolTip">
    <string>Fit tree to tree view (Ctrl+0)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+0</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="zoomInAction">
   <property name="icon">
    <iconset resource="../resources/dact.qrc">
     <normaloff>:/icons/viewmag+.png</normaloff>:/icons/viewmag+.png</iconset>
   </property>
   <property name="text">
    <string>Zoom in</string>
   </property>
   <property name="toolTip">
    <string>Zoom in on tree (Ctrl+=)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+=</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="zoomOutAction">
   <property name="icon">
    <iconset resource="../resources/dact.qrc">
     <normaloff>:/icons/viewmag-.png</normaloff>:/icons/viewmag-.png</iconset>
   </property>
   <property name="text">
    <string>Zoom out</string>
   </property>
   <property name="toolTip">
    <string>Zoom out (Ctrl+-)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="nextTreeNodeAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/dact.qrc">
     <normaloff>:/icons/1rightarrow.png</normaloff>:/icons/1rightarrow.png</iconset>
   </property>
   <property name="text">
    <string>Next node</string>
   </property>
   <property name="toolTip">
    <string>Next node (Ctrl+Right)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Right</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="previousTreeNodeAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/dact.qrc">
     <normaloff>:/icons/1leftarrow.png</normaloff>:/icons/1leftarrow.png</iconset>
   </property>
   <property name="text">
    <string>Previous node</string>
   </property>
   <property name="toolTip">
    <string>Previous node (Ctrl+Left)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Left</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="showMacrosWindow">
   <property name="text">
    <string>Macros…</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="showFilterWindow">
   <property name="text">
    <string>Bracketed sentences…</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="showStatisticsWindow">
   <property name="text">
    <string>Statistics…</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="nextAction">
   <property name="icon">
    <iconset resource="../resources/dact.qrc">
     <normaloff>:/icons/next.png</normaloff>:/icons/next.png</iconset>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
   <property name="toolTip">
    <string>Next sentence/tree (Ctrl+Down)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Down</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="previousAction">
   <property name="icon">
    <iconset resource="../resources/dact.qrc">
     <normaloff>:/icons/previous.png</normaloff>:/icons/previous.png</iconset>
   </property>
   <property name="text">
    <string>Previous</string>
   </property>
   <property name="toolTip">
    <string>Previous sentence/tree (Ctrl+Up)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Up</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="openAction">
   <property name="icon">
    <iconset resource="../resources/dact.qrc">
     <normaloff>:/icons/fileopen.png</normaloff>:/icons/fileopen.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open…</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="printAction">
   <property name="icon">
    <iconset resource="../resources/dact.qrc">
     <normaloff>:/icons/fileprint.png</normaloff>:/icons/fileprint.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Print…</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="pdfExportAction">
   <property name="icon">
    <iconset resource="../resources/dact.qrc">
     <normaloff>:/icons/pdf.png</normaloff>:/icons/pdf.png</iconset>
   </property>
   <property name="text">
    <string>Export to PDF…</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="aboutAction">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="helpAction">
   <property name="text">
    <string>Help</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="openDirectoryAction">
   <property name="text">
    <string>Open Directory…</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="saveCorpus">
   <property name="text">
    <string>Export to corpus…</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="preferencesAction">
   <property name="text">
    <string>Preferences</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+,</string>
   </property>
  </action>
  <action name="focusFilterAction">
   <property name="text">
    <string>Filter</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="focusHighlightAction">
   <property name="text">
    <string>Highlight</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+H</string>
   </property>
  </action>
  <action name="downloadAction">
   <property name="text">
    <string>Download corpus...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>DactTreeView</class>
   <extends>QWidget</extends>
   <header>DactTreeView.hh</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/dact.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>quitAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>394</x>
     <y>329</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
